launchable/utils/tracking.py

```diff
@@ -0,0 +1,98 @@
+from enum import Enum
+from typing import Dict, Any, Optional, Union
+
+from requests import Session
+from launchable.utils.http_client import _HttpClient, _join_paths
+
+from launchable.version import __version__
+
+
+class Tracking:
+    # General events
+    class Event(Enum):
+        SHALLOW_CLONE = 'shallow_clone'  # this event is an example
+
+    # Error events
+    class ExceptionEvent(Enum):
```

Comment:
Is ErrorEvent better than ExceptionEvent??

WDYT??

Reply:
[c532a15](https://github.com/launchableinc/cli/pull/594/commits/c532a15a3ab48067a10c308704423f23ce34c06f)